src/MspFcComms.cpp

```diff
@@ -60,15 +61,22 @@ namespace FcComms
     }
 
     // Send the rc commands to the FC using the member array of rc values.
-    FcCommsReturns MspFcComms::getRawRC()
+    FcCommsReturns MspFcComms::getRawRC(uint16_t rc_values[18]) //To do- add array to be populated
```

Comment:
Is this what you meant by "get the rc values after you send the message to retrieve them from the FC."?